icons/database-backup.svg

```diff
@@ -10,9 +10,10 @@
   stroke-linejoin="round"
 >
   <ellipse cx="12" cy="5" rx="9" ry="3" />
-  <path d="M3 12c0 1.18 2.03 2.2 5 2.7" />
-  <path d="M21 5v4.5" />
-  <path d="m12 16 1.27-1.35a4.75 4.75 0 1 1 .41 5.74" />
-  <path d="M12 12v4h4" />
-  <path d="M3 5v14c0 1.43 2.97 2.63 7 2.93" />
+  <path d="M7 14.5C4.6 14 3 13 3 12" />
+  <path d="M9.5 21.9C5.7 21.5 3 20.4 3 19" />
+  <path d="M3 19V5" />
+  <path d="M21 5v4" />
+  <path d="m11 16 2.5-2.5c1-1 2.3-1.5 3.5-1.5 2.8 0 5 2.2 5 5s-2.2 5-5 5c-1.6 0-3.1-.8-4-2" />
+  <path d="M11 12v4h4" />
```

Comment:
One more thing: can you merge back the two paths that became separate? (the left side and the bottom)